scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtRunner.scala

```diff
@@ -87,10 +87,7 @@ object ScalafmtRunner {
     val scala213 = Scala213
       .withAllowTrailingCommas(true)
     val default = scala213
-      .withAllowOrTypes(true) // New feature in Dotty
-      .withAllowAndTypes(true) // New feature in Dotty
-      .withAllowTraitParameters(true) // New feature in Dotty
-      .withAllowColonForExtractorVarargs(true) // New feature in Dotty
+    val Scala3 = Dotty
```

Comment:
Just a side note - we might never be able to cover all dialects with scalafmt. Scala 3 has the new keywords such as `given`, which would break the Scala 2 code.

Reply:
@tgodzik can you explain this, please? i thought that `given` wouldn't be parsed unless the dialect has the flag confirming support for this keyword. how would that break scala2?

Reply:
Sorry! I just meant that if we would enable everything in the default dialect this would be an issue. Having two separate dialect is the way to go. Dialect for Scala 2 has givens disabled and tokenizes it as a normal ident, while Scala 3 dialect will think it's a keyword. And because of that we cannot have a common dialect for both.

Reply:
ah, naturally! and in cases of a mixed codebase, we have `fileOverride` configuration parameter.